src/annotator/config/settings.ts

```diff
@@ -99,16 +103,29 @@ export function settingsFrom(window_: Window): SettingsGetters {
     }
   }
 
+  function sideBySide(): SideBySideOptions {
+    const value = hostPageSetting('sideBySide');
+
+    return {
+      mode:
+        // The sideBySide prop has to be an object.
+        // The mode prop has to be 'auto' or 'manual', if defined.
```

Comment:
The code below seems clear enough to me that these comments can be omitted.

Reply:
That's actually very true 😅 

There were more logic here at first, with comments and early returns, but I ended up extracting it to helper functions. Those have descriptive enough names so that the comments are indeed superfluous now.